qiskit_braket_provider/providers/adapter.py

```diff
@@ -422,11 +360,12 @@ def convert_qiskit_to_braket_circuit(circuit: QuantumCircuit) -> Circuit:
 
     quantum_circuit = Circuit()
     if not (
-        {gate.name for gate, _, _ in circuit.data}.issubset(translatable_qiskit_gates)
+        {gate.name for gate, _, _ in circuit.data}.issubset(
+            translatable_qiskit_gate_names
+        )
     ):
-        circuit = transpile(circuit, basis_gates=translatable_qiskit_gates)
-    if circuit.global_phase > _EPS:
-        warnings.warn("Circuit transpilation resulted in global phase shift")
+        circuit = transpile(circuit, basis_gates=translatable_qiskit_gate_names)
```

Comment:
I think you should add `optimization_level=0` to this, to avoid making changes before the user requests them (as much as possible)